vita3k/io/src/io.cpp

```diff
@@ -164,6 +164,10 @@ bool find_case_isens_path(IOState &io, VitaIoDevice &device, const fs::path &tra
         final_path = system_path.string().substr(0, system_path.string().find(addcont_id)) + addcont_id;
         break;
     }
+    case +VitaIoDevice::vs0: {
+        final_path = system_path.string().substr(0, system_path.string().find_last_of('/'));
+        break;
```

Comment:
This is not entirely accurate, this works if only the file name can be case insensitive (the folder it is in needs to have the correct casing with this approach). This is fine for me, but add a comment specifying it.

Reply:
I mainly copied what IO had and yeah, its not super great but so far only the filenames have been in the wrong case, ill add the comment on both places when i get home

Reply:
Sometimes devs put stuff in their own folders inside app0 and still have the wrong case, (`app0:resources/resource.bla`) and "resources" is case sensitive, same for addcont0 where devs decide randomly how to store their DLCs like this mess:
![image](https://github.com/Vita3K/Vita3K/assets/30161277/914a3404-04e2-489a-a2fc-28676f8d86ee)

Reply:
I don't see what you are saying has to do with my comment. 
If case isensitive paths are enabled, a file app0:folder/file.txt exists and the user asks for app0:fOLder/fILe.txt, it will resolve to the former file correctly.
However, with your approach, if a file vs0:folder/file.txt exists and the user asks for vs0:fOLder/fILe.txt, it won't work (but vs0:folder/fILe.txt will resolve correctly).
Also it looks like you mixed sensitive and insensitive in your comment.